lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3137,7 +3137,7 @@ namespace Js
 
         } autoRestore(this->GetThreadContext());
 
-        if (!Js::Configuration::Global.EnableJitInDebugMode())
+        if (Js::Configuration::Global.EnableJitInDebugMode())
```

Comment:
JIT is enabled by default for debug. The problem is background worker is processing  on updated entity while all the attach/detach happens. Reading the comments around code I have edited, IMHO the fix does the right thing and prevents this?